test/model/src/core/kv_table.cpp

```diff
@@ -161,13 +176,7 @@ int
 kv_table::insert(
   const data_value &key, const data_value &value, timestamp_t timestamp, bool overwrite)
 {
-    try {
-        item(key).add_update(
-          std::move(kv_update(value, fix_timestamp(timestamp))), false, !overwrite);
-        return 0;
-    } catch (wiredtiger_exception &e) {
-        return e.error();
-    }
+    with_transaction(return insert(txn, key, value, overwrite), timestamp);
```

Comment:
Would it be more user-friendly to have:
```suggestion
    return with_transaction(insert(txn, key, value, overwrite), timestamp);
```

Reply:
It might, but I'm not sure how to implement it. I was playing around with different interface options, but i couldn't figure out a good way to declare the variables that the macro needs to declare for any better-looking options.